rules/BUILD.bazel

```diff
@@ -126,101 +87,72 @@ bzl_library(
     srcs = ["library.bzl"],
     tags = ["manual"],
     visibility = ["//visibility:public"],
-    deps = [
-        ":hmap",
-        ":import_middleman",
-        ":precompiled_apple_resource_bundle",
-        ":utils.bzl",
-        "//rules/framework:vfs_overlay",
-        "//rules/library:resources",
-        "//rules/library:xcconfig",
-        "@bazel_skylib//lib:paths",
-        "@bazel_skylib//lib:selects",
-        "@bazel_skylib//lib:sets",
-        "@bazel_skylib//rules:write_file",
-        "@build_bazel_rules_apple//apple",
-        "@build_bazel_rules_apple//apple:apple_static_library",
-        "@build_bazel_rules_apple//apple:resources",
-        "@build_bazel_rules_swift//swift",
-        "@rules_apple_api//:api",
-    ],
+    deps = [":bzl_srcs"],
 )
 
 bzl_library(
     name = "framework",
     srcs = ["framework.bzl"],
     tags = ["manual"],
     visibility = ["//visibility:public"],
-    deps = [
-        ":features",
-        ":force_load_direct_deps",
-        ":library",
-        ":plists",
-        ":providers",
-        ":transition_support",
-        "//rules/framework:vfs_overlay",
-        "//rules/internal:objc_provider_utils",
-        "@bazel_skylib//lib:partial",
-        "@bazel_skylib//lib:paths",
-        "@bazel_tools//tools/cpp:toolchain_utils.bzl",
-        "@build_bazel_rules_apple//apple",
-        "@build_bazel_rules_swift//swift",
-        "@rules_apple_api//:api",
-    ],
+    deps = [":bzl_srcs"],
 )
 
 bzl_library(
     name = "apple_patched",
     srcs = ["apple_patched.bzl"],
     tags = ["manual"],
     visibility = ["//visibility:public"],
-    deps = [
-        ":features",
-        ":providers",
-        "//rules/framework:vfs_overlay",
-        "@build_bazel_rules_apple//apple",
-        "@build_bazel_rules_swift//swift",
-    ],
+    deps = [":bzl_srcs"],
 )
 
 bzl_library(
     name = "app_clip",
     srcs = ["app_clip.bzl"],
     tags = ["manual"],
     visibility = ["//visibility:public"],
-    deps = [
-        ":force_load_direct_deps",
-        ":plists",
-        "//rules/internal:framework_middleman",
-        "@build_bazel_rules_apple//apple:ios",
-    ],
+    deps = [":bzl_srcs"],
 )
 
 bzl_library(
     name = "app",
     srcs = ["app.bzl"],
     tags = ["manual"],
     visibility = ["//visibility:public"],
-    deps = [
-        ":force_load_direct_deps",
-        ":library",
-        ":plists",
-        "//rules/internal:framework_middleman",
-        "@build_bazel_rules_apple//apple:ios",
-    ],
+    deps = [":bzl_srcs"],
 )
 
 bzl_library(
     name = "extension",
     srcs = ["extension.bzl"],
     tags = ["manual"],
     visibility = ["//visibility:public"],
+    deps = [":bzl_srcs"],
+)
+
+# A library for all of the bzl_srcs, to satisfy `stardoc`'s deps attribute.
+bzl_library(
+    name = "bzl_srcs",
```

Comment:
Would `name = "bzl_deps"` better reflect how this is used?

I'm not a stardoc expert, but is it an anti-pattern to just include the world here instead of being more targeted in the specific `bzl_library`s above?  Maybe we keep this to all external deps and leave in-repo ones on the individual libraries?